### ISSUE TITLE
adapter.js: Uncaught (in promise) DOMException: RTCPeerConnection is gone (did you enter Offline mode?)

### ISSUE THREAD

**Issue by paulmenzel** [bigbluebutton/bigbluebutton#10719]
Using BigBlueButton 2.2.28, 

**Describe the bug**
Using BigBlueButton 2.2.28, Firefox reports the error below, when closing the echo test window, from `adapter.js:1869`:

    Uncaught (in promise) DOMException: RTCPeerConnection is gone (did you enter Offline mode?)

**To Reproduce**
1. Open the console of the browser developer tools.
2. Join a room with microphone.
3. Wait until the echo test window is shown.
4. Close the window by pressing the x on the top right.
5. See error.

**Expected behavior**
No errors should be shown in the browser console.

**Actual behavior**
The error below is shown.

    Uncaught (in promise) DOMException: RTCPeerConnection is gone (did you enter Offline mode?)

**BBB version (optional):**

2..2.28

**Comment by ffdixon** [comment]
Hmm ... can you trigger this exception when using https://test.bigbluebutton.org/?

**Comment by paulmenzel** [comment]
Yes, I can. Though not the first time closing the echo test window. I had to try it several times, and often it happened when disabling/exiting the audio by clicking on the blue headset button.
